### DISCUSSION TITLE
Allow disabling status bar transparency

### DISCUSSION THREAD

**Discussion by mihic** [plateaukao/einkbro#251] [Ideas]
Currently the status bar tries to match the color of the webpage below. This causes annoying flicker when scrolling the webpage (even when using the page scroll functions) as the status bar changes color slightly and that causes e-ink to redraw the whole status bar. 
It would be nice to have the status bar set as a fixed color, that is unaffected by the webpage displayed "under" it.

**Comment by plateaukao** [comment]
I don't know how to make statusbar color fixed; so I added an option to hide statusbar instead. You could try latest version when it's released.
